simulators/README.md

```diff
@@ -101,7 +101,7 @@ Note: A profile will be created and activated only if no other profiles are alre
 
 ### Deployment
 
-To deploy this project, upload the zip file to the Cumulocity as Microservice. The zip file can be created locally as described above or downloaded from the [releases](releases) section.
+To deploy this project, upload the zip file to the Cumulocity as Microservice. The zip file can be created locally as described above or downloaded from the [releases](https://github.com/SoftwareAG/oee-simulators/releases) section.
```

Comment:
why is an absolute URL required here?

Reply:
Because there is no release directory in the project so the link can not be redirected and will show issues when users read documents and click. Add absolute URL here will connect user to the releases page on github and they can download them during the study of document.